etl/steps/export/multidim/education/latest/proficiency.py

```diff
@@ -0,0 +1,393 @@
+        "reading_desc": "identify key ideas in texts and connect them to their own experiences",
+    },
+    "lower_secondary": {
+        "keywords": ["prepfuture_2", "lowersec"],
+        "display_name": "Lower secondary education",
+        "title_term": "lower secondary school age",
+        "math_desc": "demonstrate skills in computation, application problems, matching tables and graphs, and making use of algebraic representations",
+        "reading_desc": "connect main ideas across different text types, understand the author's intentions, and draw conclusions from the text",
+    },
+}
+
+# Subject configurations
+SUBJECTS = {
+    "mathematics": {
+        "keywords": "math",
+        "display_name": "Mathematics",
+        "title_term": "mathematics",
+    },
+    "reading": {
+        "keywords": "read",
+        "display_name": "Reading",
+        "title_term": "reading",
+    },
+}
+
+# Gender configurations
+GENDERS = {
+    "both": "children",
+    "male": "boys",
+    "female": "girls",
+    "sex_side_by_side": "boys and girls",
+}
+
+# Population configurations
+POPULATIONS = {
+    "all_children": {
+        "title": "all children",
+        "context": "This data accounts for school completion rates to estimate skills for all children, not just those in school.",
+    },
+    "students": {
+        "title": "students",
+        "context": "This data includes only students enrolled in school.",
+    },
+}
+
+# Dimension mapping configurations
+LEVEL_KEYWORDS = {}
+for level_key, config in EDUCATION_LEVELS.items():
+    for keyword in config["keywords"]:
+        LEVEL_KEYWORDS[keyword] = level_key
+
+SUBJECT_KEYWORDS = {config["keywords"]: key for key, config in SUBJECTS.items()}
+SEX_KEYWORDS = {"both_sexes": "both", "male": "male", "female": "female"}
+
+# Exclusion patterns for column filtering
+EXCLUSION_PATTERNS = [
+    "urban",
+    "rural",
+    "the_global_age_specific_literacy_projections_model",
+    "poorest_quintile",
+    "richest_quintile",
+    "adjusted",
+    "very_affluent",
+    "very_poor",
+    "immigrant",
+    "native",
+    "language_of_the_test",
+    "grade_2",
+    "grade_3",
+]
+
+
+def run() -> None:
+    """Main function to process prepared for the future data and create collection."""
+    # Load inputs
+    config = paths.load_collection_config()
+    ds = paths.load_dataset("education_sdgs")
+    tb = ds.read("education_sdgs", load_data=False)
+
+    # Filter both prepared for the future and student proficiency columns
+    proficiency_cols = get_proficiency_columns(tb)
+
+    # Select only relevant columns
+    tb = tb.loc[:, ["country", "year"] + proficiency_cols].copy()
+
+    # Adjust dimensions
+    tb = adjust_dimensions(tb)
+
+    # Create collection
+    c = paths.create_collection(
+        config=config,
+        tb=tb,
+        common_view_config=MULTIDIM_CONFIG,
+    )
```

Comment:
I note that you often use `adjust_dimensions`. To be clear this is totally fine, so no action required!

I'm just saying, in case it has been lost in translation. The preferred and initial idea with dimensions is that they are defined in garden by indexing the tables with appropriate columns (not only `country` and `year`).

I reckon sometimes is tricky, but I think it is generally nice to do there if possible.